pkg/arvo/app/contact-hook.hoon

```diff
@@ -242,7 +247,10 @@
         %delete
       =.  synced  (~(del by synced) path.fact)
       :_  state
-      [(group-poke [%unbundle path.fact])]~
+      :~  (group-poke [%unbundle path.fact])
+          (metadata-hook-poke [%remove path.fact]) 
+          (metadata-poke [%remove path.fact [%contacts path.fact]])
+      ==
```

Comment:
Can you explain why, for (local) creation, metadata management lives in the view, but for deletion, it happens in the hook?

Reply:
Within contacts, we’ve followed a pattern in which:
- %contact-view sets everything up and has a delete action that cleans everything up
- alternatively, if you delete the group in group-store that contacts depends on, %contact-hook will clean up all of the associated data